lib/txgh/github_repo.rb

```diff
@@ -19,6 +19,22 @@ def process_all_branches?
       branch == 'all'
     end
 
+    def should_process_branch?(candidate)
+      process_all_branches? ||
+        candidate.include?(github_config_branch) ||
+        candidate.include?('L10N')
```

Comment:
might be good to document what's so special about this particular branch name/assign it to a constant